frontend/src/scenes/billing/billing-utils.ts

```diff
@@ -222,3 +222,32 @@ export const convertLargeNumberToWords = (
         !previousNum && multipleTiers ? ` ${productType}s/mo` : ''
     }`
 }
+
+export const getProration = ({
```

Comment:
I actually think this function should just take dates and do the calculation for time remaining and time total. Otherwise there's a risk that those calculations could differ if used elsewhere.

I don't know where else this might be used, so this is probably an unnecessary optimization, but was just a thought I had.

Reply:
What do you mean by:
> Otherwise there's a risk that those calculations could differ if used elsewhere.

This function is meant to be the business logic for the calculation.